### DISCUSSION TITLE
DistributedSampler

### DISCUSSION THREAD

**Discussion by sakerlrf** [open-mmlab/mmaction2#1177] [Q&A] [answered]
Hello,

When training a model with RawVideoFrames, I need to make sure that there is:
- **sufficient shuffling of the data of the data**

How could I work this around either on the **_epoch_** or the **_batch level_**?
Does **_"DistributedSampler"_** do such kind of shufflings?

If yes, **_how_** can I use this for training data _in the main model config_ script?

Thank you!

**Comment by dreamerlin** [comment, marked as answer]
In `class SampleFrames`, we make sure to sufficiently explore and exploit sampling frames
